### ISSUE TITLE
Helm Chart: Add Support for Deploying Additional Manifests (e.g., Role and RoleBinding) in cert-exporter

### ISSUE THREAD

**Issue by Mistral-valaise** [joe-elliott/cert-exporter#172]
I would like to propose an enhancement to the cert-exporter Helm chart to include support for deploying additional Kubernetes manifests, such as Role and RoleBinding, using a configurable option. This feature would allow users to specify custom manifests in their values.yaml file, which would then be deployed as part of the Helm chart installation.

Proposed Solution:

1. Create a New Template File:

Add a new file named **extra-manifests.yaml** in the **templates/cert-manager/extra-manifests.yaml** directory.
The content of the file would be as follows:
```yaml
{{ range .Values.extraManifests }}
---
{{ tpl . $ }}
{{ end }}
```
2. Update values.yaml:
Introduce a new configuration option extraManifests in the values.yaml file:
```yaml
...
# Extra manifests to deploy as an array
extraManifests: []
  # - |
  #   apiVersion: rbac.authorization.k8s.io/v1
  #   kind: Role
  #   metadata:
  #     name: cert-exporter
  #   rules:
  #     - apiGroups:
  #         - ""
  #       resources:
  #         - configMaps
  #         - secrets
  #       verbs:
  #         - get
  #         - watch
  #         - list
  # - |
  #   apiVersion: rbac.authorization.k8s.io/v1
  #   kind: RoleBinding
  #   metadata:
  #     name: cert-exporter
  #   roleRef:
  #     apiGroups: rbac.authorization.k8s.io/v1
  #     kind: Role
  #     name: cert-exporter
  #   subjects:
  #     - kind: serviceAccount
  #       name: cert-exporter
...
```

Use Case:

This feature would be particularly useful for users who need to deploy additional Kubernetes resources, such as Roles and RoleBindings, in conjunction with cert-exporter. By allowing these to be configured and deployed through the Helm chart, it simplifies the deployment process and increases the flexibility of the chart.

Contribution:
I am willing to implement this feature and submit a pull request. Please let me know if this enhancement would be welcome, or if there are any additional considerations or guidelines I should follow while working on this.

Thank you for your time and consideration.

**Comment by joe-elliott** [comment]
yeah, i like this idea. instead of building every feature imaginable the helm chart would just be extendable by anyone to add the manifests they need
